tests/unit/create/test_creator.py

```diff
@@ -398,6 +398,10 @@ def test_create_long_path(tmp_path):
     subprocess.check_call([str(result.creator.script("pip")), "--version"])
 
 
+@pytest.mark.skipif(
+    sys.version_info[:2] == (3, 8),
+    reason="Disable on Python 3.8, which still uses pip 25.0.1 (without https://github.com/pypa/pip/pull/13330)",
+)
```

Comment:
Tests are passing due to skipping this on 3.8.

We could continue to bundle `wheel` only on 3.8, instead? That would defer some of the cleanup here until after 3.8 goes out of support in virtualenv.

Reply:
Yes.